app/controllers/login.js

```diff
@@ -1,5 +1,7 @@
-import Ember from 'ember';
+import { Controller, computed } from 'ember';
 
 import OsfTokenLoginControllerMixin from 'ember-osf/mixins/osf-token-login-controller';
 
-export default Ember.Controller.extend(OsfTokenLoginControllerMixin);
+export default Controller.extend(OsfTokenLoginControllerMixin, {
+    login: computed.alias('model'),
```

Comment:
If you believe routable components will be a thing, then it would be feasible to keep. I've downgraded it to just a warning for the [COS config](https://github.com/CenterForOpenScience/javascript/blob/develop/packages/eslint-config-cos-ember/index.js#L21), so it won't make anything fail, but I'd be fine with just nixing it altogether. Your call, I don't mind either way.